### DISCUSSION TITLE
VS2022 Copilot asks me for github code but does not provide it

### DISCUSSION THREAD

**Discussion by RobertIonutF** [community/community#51967] [Copilot Conversations]
### Select Topic Area

Question

### Body

So I'm trying to setup copilot for VS 2022.  I have windows 11, I have installed the extension and I do have the monthly subscription plan activated, however, after I click the ok button to lead me to Copilot auth, nothing happens. I have tried reinstalling it multiple times, but it doesn't work, it worked for Visual Studio Code but here, it doesn't want to.

It wll only show these options when I click for it: 
![image](https://user-images.githubusercontent.com/56986332/229846803-8b5d080f-1eee-454c-bbc4-c34bf372c743.png)

You can see in the console that it says Telemtry initialized but it doesn't work and it doesn't lead me to the auth page.

**Comment by ayim** [comment]
Hey @robertlonutF, does clicking Sign In show the dialog that has the "OK" button?
The output window should show the code and URL at that point.
Feel free to try it and let me know, if this doesn't work, we'd love to do a screenshare with you and diagnose the problem.

- ayim at github dot com
